res/genesis/README.md

```diff
@@ -0,0 +1,6 @@
+# Information about generating spec files
+
+- Developers should continue to generate specs files locally for development and testing.
+- To generate spec files locally run ```./scripts/generate_specs.sh 2000 true```, Once MRC collator is built. Refer to MRC readme for more details.
+- Additional steps in main [job](https://github.com/LibertyDSNP/mrc/actions/workflows/main.yml?query=branch%3Amain) is run to generate the spec files on the main branch.
+- Latest spec files can be downloaded from [build artifacts](https://github.com/LibertyDSNP/mrc/actions/workflows/main.yml?query=branch%3Amain).
```

Comment:
Generic placeholder file, remove the specs since developers can generate it locally